### ISSUE TITLE
[IGNORE, DUPLICATE OF #82] As a member who want to edit and delete, I want to utilise the action of buttons so that edit and delete keys can be used.

### ISSUE THREAD

**Issue by vikasithsai** [cse442-at-ub/project_s23-cinco#72]
Acceptance Test
Test 1:

A static page with the option to display "Edit" and  "Delete".
Though it's a static page with no functionality the keys doesn't work. Also it's on running on the local host

**Comment by junqiwu02** [comment]
There are grammatical issues in the title. and all stories must be a blocked by dependency. *Please read the style guide*
